socialcalcconstants.js

```diff
@@ -199,7 +211,7 @@ SocialCalc.Constants = {
 
    //*** SocialCalc.TableEditor
 
-   defaultImagePrefix: "images/sc_", // URL prefix for images (e.g., "/images/sc")
+   defaultImagePrefix: "/images/sc_", // URL prefix for images (e.g., "/images/sc")
```

Comment:
Some people run EtherCalc not at `/` but relatively — I would prefer that we keep this unless there's practical problems